indico/modules/events/registration/templates/display/event_header.html

```diff
@@ -0,0 +1,48 @@
+<tr>
+    <td class="leftCol">{% trans %}Registration{% endtrans %}</td>
+    <td>
+        {% for regform in regforms %}
+            <div class="event-service-row">
+                <div class="event-service-row-collapsed clearfix">
+                    <div class="event-service-info left">
+                        <i class="icon-ticket"></i>
+                        <span class="event-service-title">
+                            {% if loop.length == 1%}
+                                <em>{% trans %}This event is open to new participants.{% endtrans %}</em>
+                            {% else %}
+                                {{ regform.title }}
+                            {% endif %}
+                        </span>
+                    </div>
+                    <div class="event-service-toolbar toolbar right">
+                        <div class="group">
+                           {% set registration = session.user and regform.get_registration(user=session.user) %}
+                           {% if registration  %}
+                                <a class="i-button i-button-small icon-edit highlight"
+                                    href="{{ url_for('event_registration.display_regform', regform) }}">
+                                    <strong>{% trans %}Modify registration{% endtrans %}</strong>
+                                </a>
+                           {% elif regform.limit_reached %}
+                               <a class="i-button i-button-small highlight disabled"
+                                  title="{% trans %}There are no more places available.{% endtrans %}"
+                                  href="#">
```

Comment:
if you remove the `href` you don't get an ugly `#` in the url when clicking anyway